ci/jenkins/pipelines/prs/skuba-code-lint.Jenkinsfile

```
@@ -13,9 +13,27 @@ pipeline {
     }
 
     stages {
-        stage('Collaborator Check') { steps {
-            sh(script: "${PR_MANAGER} check-pr --collab-check", label: "Checking if collaborator")
-        }}
+        stage('Collaborator Check') { steps { script {
+            def membersResponse = httpRequest(
+                url: "https://api.github.com/repos/SUSE/skuba/collaborators/${CHANGE_AUTHOR}",
+                authentication: 'github-token',
+                validResponseCodes: "204:404")
+
+            if (membersResponse.status == 204) {
+                echo "Test execution for collaborator ${CHANGE_AUTHOR} allowed"
+
+            } else {
+                def allowExecution = input(id: 'userInput', message: "Change author is not a SUSE member: ${CHANGE_AUTHOR}", parameters: [
```

Should this message say "skuba" rather than SUSE, for "why isn't my PR running" debugging purposes?  For example:
```suggestion
                def allowExecution = input(id: 'userInput', message: "Change author is not a member of the skuba repository: ${CHANGE_AUTHOR}", parameters: [
```